README.md

```diff
@@ -286,14 +286,6 @@ invokes cmake commands as needed.
     make debug
     ```
 
-* **Optional**: to build statically-linked binaries:
```

Comment:
Why was this section removed ?

Reply:
I originally had the same thought, but, if you look below, `make release-static` is now the default build instruction, so this is redundant.

Reply:
I removed this section because it is misleading. `release-static` will not produce statically-linked binaries on Linux. To statically link dependencies (other than libc) on Linux, `make depends` should be used, which is documented under the "Cross compilation" section.

Static builds for Linux are [not yet](https://github.com/monero-project/monero/blob/master/CMakeLists.txt#L1012) supported, however work is underway (#9207).